yesod-core/Yesod/Core/Class/Yesod.hs

```diff
@@ -570,7 +575,7 @@ widgetToPageContent w = do
                 ^{mkScriptTag s}
             $maybe j <- jscript
                 $maybe s <- jsLoc
-                    <script src="#{s}">
+                    ^{mkDefaultScriptTag s (jsAttributes master)}
```

Comment:
You can use the `*{..}` syntax for multiple attributes, described at: http://www.yesodweb.com/book/shakespearean-templates#shakespearean-templates_attributes